x-pack/plugins/security_solution/public/cloud_defend/links.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { getSecuritySolutionLink } from '@kbn/cloud-defend-plugin/public';
+import { i18n } from '@kbn/i18n';
+import type { SecurityPageName } from '../../common/constants';
+import { SERVER_APP_ID } from '../../common/constants';
+import type { LinkItem } from '../common/links/types';
+import { IconCloudDefend } from '../management/icons/cloud_defend';
+
+const commonLinkProperties: Partial<LinkItem> = {
+  hideTimeline: true,
+  capabilities: [`${SERVER_APP_ID}.show`],
+};
+
+export const rootLinks: LinkItem = {
+  ...getSecuritySolutionLink<SecurityPageName>('policies'),
+  globalNavPosition: 3,
+  ...commonLinkProperties,
+};
+
+export const manageLinks: LinkItem = {
+  ...getSecuritySolutionLink<SecurityPageName>('policies'),
+  description: i18n.translate('xpack.securitySolution.appLinks.cloudDefendPoliciesDescription', {
+    defaultMessage: 'View drift prevention policies.',
+  }),
+  landingIcon: IconCloudDefend,
+  ...commonLinkProperties,
+};
+
+// currently using the CSP category, as it's weird to have two categories each with one item..
+// saving this for when we add other pages
+/* export const manageCategories: LinkCategories = [
```

Comment:
are you OK with adding this code back in when you're ready to merge it?  I think we should generally avoid committing commented out code.